src/components/RichTextEditor/Plugins/TextStylePlugin/TextStyles/textStyles.ts

```diff
@@ -22,10 +22,16 @@ export enum TextStyles {
     ELEMENT_CUSTOM2 = 'custom2',
     ELEMENT_CUSTOM3 = 'custom3',
     ELEMENT_QUOTE = 'quote',
+}
+
+enum FixedTextStyles {
```

Comment:
this should be included in TextStyles, the separate Type could also be with Omit